src/main/resources/static/uml/Quiz.html

```diff
@@ -0,0 +1,12 @@
8nbvG54WA4tRlKo0+HKlTx+Sq/df0vvv/w127t++9XXMBvfshVs644/b4Dsdcbpny5DsgnXoRVCaPeEofaBPYBIzLwyWEXxfwDhQDIRnFKcBrOAVfZDiDVAXqyD077w93pLRJqm3+MpKsxWOg+/QrRfCIx0I8DhOKTMYTq3GA2kJZA2oMUqgzj1bAr0g3XlJSusscBh6myR4zAZMqqy9eBlEM5ymeM06wtvIRz4tFUhlhTTGXwvsSXtx4SkWZOLopUKiQNwivEZpvIcq7ClddrorxtaEEnYlj1kapa0q7AVUytuUr5dF38XrPsE+8KIlzLd4n2HWX6hb4vt0R/Y+p/46LwSMIy9FM7JiSZXh4EtlqiUpY8MjWNLWBZYcQ/EbsOXn/QYoV1B6SOMgWgpsCsufVlgyRE9pI0MmG28Bfdxlda6tkvKJTp+QMLR9CjNmWAW+j6KMWVIv9XJ+IhyywUGUZutjz+ADqzjXLu2RDQOaQ1kvy/Ah1eN0jiPgKy/IGAgBs+4QYVgJax3ete0Mt6/j2M5fzexVA/pYVA0RVQG9MMgEzCpdl3KA33wt0K4BJNIdw/Izgfp6rAt4myLepgTb0HtE4T1OgjTApP84r8th/sNgtRXFhn5AanSC1Rb1x8hwwnzS+d98UbaLFMdAWtJH7O/IMHUt+wofjeiei3c18rDD1VjBUWQFsy9WYBu6xgoFqN+2MP9sD1XF95wMJEp2xG4o6e8G1FVRn6raDU5PqNsHUd8mKP7gi5gP3NALN+i6qprvjR3MvtmBtRCtwYFR1BnFMn40o1gNjLJE6Uepqa9dDACrA+xMfjTAdgPACQ/wM4aOAN5hT3eDfGr9YMgNUfjfemskQDgEk04aTJrUYztjXWIUGtLgzuSAL9AcTNK5YJIpCV69lWCSIaoZEkxaAlvyhoiWbWSypbWRO4Ne3euKIzpuNFNqVWT2Sq2CzAT6RSWcqczozUEsKV87vQWxZDptCGJ1hFVVXE36QtWRoFqPYS2OiGERlcfFsDp2dqS3zHUBBvUXidDRLqRVP0rfJauc8JVLO07sQvquK2H08tdU67GXKM78VhD08ldI6ok6QmwZJDfftsTQIJVmGIfIi8joAr7dIOIVZYHotkhlge4afUkDt1UafCC26ZO3QAe2L3FxPqENEbqYjCpvmedW51XjYFmuZ7uJrJ3BRLa0n89Edglj14xkV5I6MdyphNMOBcybjWTbrhvJbznjyjTrYNYoiAVlUSYBXMpgvZk1UwHVpyDyZ3umtRsUdVa4C2BxakJLkHB5Z7lulqrjYzoij2b73E6RhIlbO/tF9evxTCnRrzKm7E29Mml6SNagyL8iR7egFOFM7LRoiLo6gebvAzKqvOQvEVOfKHzEu5uSMMsI8IBxBpB8L1llXRV6kY0FbzLYM1rlBY3qKcHbeIEOwUfXAjoEU1UBZzIXFXXHazuLjihGoZcGz6g2zAN67Z5wdNmtyUeaLI0TXvmUabOSU4SebKutp3xN2nsacz2ZHOPipycw7/vQmqZ4okW7vGTYl9ycWUgVMdEilB6pnZSrtxkYX8uMvec4BH+x2BGZVZgbbLqhLjHY9qPHJemwRsYR4Wa28KfhKf5kSo94GQJe6mAx2+LfitZpwDLqe804H3ZiZiGEaSTk5WmT9q/Yqjysoomo4lgVBuxhsI9ybjpyhsABTcyiriumJ2EVdj76/GJZllhuNjAeQ0zc2xqz1K2JlyD9P/kO65SX/qL1yPfrl0q1631hgcAcKo1I8a/qs7JZVirb+Wc2LExFw6JiL9oSp5XRuhoYU84sAMf9dQaGxRsYJi+sGgyMk7GhmCogHgsY/CACwhDkyxsK+fyMWVGLy1JK06KmND/Os8rPlxY1xZQHO2P/gOLngEgX4nB/rBB+UR+72Mhd0pRyPusrTWmL8TyC7t3vtzSMgpdLiQAaAFU/ZX9eQE1ZTmII0HaE1RXPS0lh7StAa8rC7iSrGAZJeqEeP62dm5ztP/hC+rGQ5UKrb1sUFwHhpGNEuNJlHhZOOsWF8+4WMQJVmB0OLOdxaEqwEF0iybLe2HnFhiOKQ2z6JPtxYqjtR93sbUeKKRMBvorvSAPCza4jcwP1qhNYFJqcQKnDWfiYDQ5n1XGshbAbTelWL5Ih2O5FUpDeiBdpu7zvZ7/Si3S4gNe4OE/b4kUCh3j7SjW6/ZqHzEfWWfaw5Oa8y5N6AcxH5nzUwuBXEGaD03pip5U7pjDRbYlQtCW2p+u27pN/udNqiQkX5rSSkzalSVA9d/NrKt9iZ3dzWmV81pvTKg9JoBCtYa61qMRNnTZg3MWPPSvGoh87BCY6ACr1YM8JKLtGOAQmTgmr5BCsFNa+3CCr6RJv7ovT+/yq/njZCCVZs8xTPugzv6ue9ea6O2NwJB/5b2hHB7/A6w2xbvImc1bKzvM2XG0uemy43HiicAytdqaQzCkO7DV22S3WIpzP/wNEi3+BH//OW74PwTdeeLFPe3pkZ/YHlaN4y1RTNBT13m5hWWKGn5qClYP5an7qcOX0dX7qhE+uupJfnijyqDWddeDy8U/ipg6XBNvljqXMcQfufkr5qy+hI7skKFUxnwfV8jqITdGbkUJsHhAh3TBWSLMec3SsFodnYX39UtOmo0pg/9JwW06TQeEexQFMkugn2Rn34rzZqHrarDx81pBqaNQErbkB5iK05gbYvnkjuQGXu5Y1dvjDPqq5AZc/q+Y4armBk+kZhVtbnZNV7Tz0mtRTe0bJeFNcY/NY2/zPsKpyjelOL6eVf5OWfk+VX+InYNJ0Qb/5pePSqUfK09fyK5XDYw0EsV0XxE5HOXwGecpu9b2RnWHy52wd/kKH8s7oT55CsfyZ7bx6+WPl5s0/&lt;/diagram&gt;&lt;/mxfile&gt;&quot;,&quot;toolbar&quot;:&quot;pages zoom layers lightbox&quot;,&quot;page&quot;:0}"></div>
```

Comment:
Interesting, what's this?

Reply:
Documentation (UML class diagram) for Quizes. Adding it to the running list of items from the [slack convo](https://ics499-team2.slack.com/archives/C02TMJNP28K/p1651114873088139)